src/intrinsics/prepack/global.js

```diff
@@ -239,8 +239,8 @@ export default function(realm: Realm): void {
     configurable: true,
   });
 
-  // Helper function used to instatiate a residual function
-  function deriveNativeFunctionValue(unsafe: boolean): NativeFunctionValue {
+  // Helper function used to instantiate a residual function
+  function deriveNativeFunctionValueForResidualCall(unsafe: boolean): NativeFunctionValue {
```

Comment:
I'm no fan of the term "derive" in general. It would be nice if we can phase out its use, starting with any method that does not ultimately call Generator.deriveAbstract or Generator.deriveConcrete.